src/index.js

```diff
@@ -0,0 +1 @@
+export * as Button from "@components/button/Button";
```

Comment:
We need to add an export for all the typography components (i.e., Title, Text, Code)

Reply:
fixed